### DISCUSSION TITLE
Lock mode

### DISCUSSION THREAD

**Discussion by yamafaktory** [ghostty-org/ghostty#4085] [Feature Requests, Ideas] [answered]
Hi folks :)

First of all, what a lovely project, can't thanks the contributors enough for this amazing piece of work!

I was wondering if introducing something like a "lock mode" as a way to lock the interface, disabling all keybindings except the one needed to "unlock"  could be an interesting feature. Basically something in the same vein as https://zellij.dev/old-documentation/keybindings-modes#locked.

Thanks!

**Comment by pluiedev** [comment, marked as answer]
Duplicate of #3575
